tests/test_messages.py

```diff
@@ -27,6 +28,54 @@ def test_balance_proof(get_random_bp):
         bp.token_network_address = '0x11e14d102DA61F1a5cA36cfa96C3B831332357b4'
 
 
+def test_balance_proof():
+    balance_proof = BalanceProof(
+        channel_identifier=ChannelIdentifier(123),
+        token_network_address=Address('0x82dd0e0eA3E84D00Cc119c46Ee22060939E5D1FC'),
+        nonce=1,
+        chain_id=321,
+        balance_hash='0x%064x' % 5,
+        transferred_amount=1,
+        locked_amount=0,
+        additional_hash='0x%064x' % 0,
+    )
+    serialized = balance_proof.serialize_data()
+
+    assert serialized['channel_identifier'] == 123
+    assert serialized['token_network_address'] == '0x82dd0e0eA3E84D00Cc119c46Ee22060939E5D1FC'
```

Comment:
why not `assert serialized['token_network_address'] == balance_proof.token_network_address`?